ui/src/app/shared/utils/array/array.utils.ts

```diff
@@ -23,7 +23,7 @@ export namespace ArrayUtils {
    * @param arr the arr
    * @returns a number if arr not empty, else null
    */
-  export function findBiggestNumber(arr: number[]): number | null {
```

Comment:
test pattern has null and undefined. so this function work with null and undefined.